services/brig/deb/opt/brig/templates/user/de/email/verification.html

```diff
@@ -1 +0,0 @@
-../../en/email/verification.html
```

Comment:
`Gib diesen Code ein, um deine e-Mail-Adresse zu überprüfen und erstelle dein Konto.` should be changed to `Gib diesen Code ein, um deine e-Mail-Adresse zu überprüfen und dein Konto zu erstellen.`